core/src/main/java/hudson/model/Computer.java

```diff
@@ -625,6 +625,10 @@ public final boolean isOnline() {
         return !isOffline();
     }
 
+    public boolean isConnected() {
```

Comment:
Missing `@Override`, and should `@inheritDoc` then explain that it is checking `getChannel`.

Reply:
ty noticing, fixed now.